prometheus/src/components/Chart/KedaMetricsChart/KedaMetricsChart.tsx

```diff
@@ -0,0 +1,192 @@
+import { Icon } from '@iconify/react';
+import { Loader, SectionBox } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+import { useCluster } from '@kinvolk/headlamp-plugin/lib/k8s';
+import Alert from '@mui/material/Alert';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import ListSubheader from '@mui/material/ListSubheader';
+import MenuItem from '@mui/material/MenuItem';
+import Paper from '@mui/material/Paper';
+import Select from '@mui/material/Select';
+import { useEffect, useState } from 'react';
+import {
+  getConfigStore,
+  getPrometheusInterval,
+  getPrometheusPrefix,
+  getPrometheusResolution,
+  getPrometheusSubPath,
+} from '../../../util';
+import { PrometheusNotFoundBanner } from '../common';
+import { KedaScalerChart } from '../KedaScalerChart/KedaScalerChart';
+
+/**
+ * Props for the KedaMetricsChart component
+ * @interface KedaMetricsChartProps
+ * @property {string} scalerMetricsQuery - The Prometheus query to fetch KEDA Scaler Metrics Value
+ * @property {string} hpaReplicasQuery - The Prometheus query to fetch HPA Replicas Count
+ * @property {string} activeJobsQuery - The Prometheus query to fetch Active Jobs Count
+ * @property {number} minReplicaCount - Minimum Replica Count as specified in the YAML Spec of KEDA Resource
+ * @property {number} maxReplicaCount - Maximum Replica Count as specified in the YAML Spec of KEDA Resource
+ */
+export interface KedaMetricsChartProps {
+  scalerMetricsQuery: string;
+  hpaReplicasQuery?: string;
+  activeJobsQuery?: string;
+  minReplicaCount: number;
+  maxReplicaCount: number;
+}
+
+export function KedaMetricsChart(props: KedaMetricsChartProps) {
+  enum prometheusState {
+    UNKNOWN,
+    LOADING,
+    ERROR,
+    INSTALLED,
+  }
+
+  const cluster = useCluster();
+  const configStore = getConfigStore();
+  const useClusterConfig = configStore.useConfig();
+  const clusterConfig = useClusterConfig();
+
+  const [refresh, setRefresh] = useState<boolean>(true);
+  const [prometheusPrefix, setPrometheusPrefix] = useState<string | null>(null);
+  const [state, setState] = useState<prometheusState>(prometheusState.LOADING);
+  const [isVisible, setIsVisible] = useState<boolean>(false);
+
+  useEffect(() => {
+    const isEnabled = clusterConfig?.[cluster]?.isMetricsEnabled || false;
+    setIsVisible(isEnabled);
+
+    console.log('clusterConfig', clusterConfig);
+    console.log('isEnabled', isEnabled);
```

Comment:
Please remove the console logs used for dev purposes

Reply:
Hi @yolossn, these logs were also present in `DiskMetricsChart.tsx` and `GenericMetricsChart.tsx`, so I thought they are there for some purpose?

Reply:
Nope might have been there for testing purposes only, we can remove them.